graylog2-web-interface/src/components/configurations/SearchesConfig.tsx

```diff
@@ -253,6 +273,14 @@ const SearchesConfig = () => {
         </dd>
       </dl>
 
+      <dl className="deflist">
+        <dt>Cancellation timeout</dt>
+        <dd>{cancellationTimeout(viewConfig)}</dd>
+        <dd>The time in seconds per widget after which search execution will be canceled automatically.
+          That minimizes amount of executed searches and improves performance.
```

Comment:
Maybe let's say `This minimizes the amount of executed searches and improves performance`?